packages/pluggableWidgets/intro-screen-native/src/SwipeableContainer.tsx

```diff
@@ -293,7 +293,7 @@ export const SwipeableContainer = (props: SwipeableContainerProps): ReactElement
             setActiveIndex(newIndex);
             onSlideChange(newIndex, lastIndex);
         },
-        [activeIndex, width]
+        [activeIndex, width, rtlSafeIndex, onSlideChange]
```

Comment:
@UrazAkgultan @ablok I think adding `rtlSafeIndex` and `onSlideChange` will make this function change with every render which makes the `useCallback` hook useless. you can use `useCallback` for both functions as well.

Reply:
Fixed